rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -22,29 +22,30 @@ servers:
   - url: ${rest.server.url}
 
 paths:
-  /index/{Id}:
+  /index/{datasetId}:
```

Comment:
I'm not sure if this change requires a version bump.

Reply:
Oh wait the parameter name changed so it probably does require a version bump.

Reply:
This is a path parameter, so I don't think it would break any clients.